services/frontend-service/pkg/config/config.go

```diff
@@ -20,6 +20,7 @@ import "time"
 
 type ServerConfig struct {
 	CdServer            string        `default:"kuberpult-cd-service:8443"`
+	TlsEnabled          bool          `default:"false" split_words:"true"`
```

Comment:
We  will need this on   the  rollout service as well. Also, I'm not too happy with the config name "TlsEnabled" could also  mean that the http server creates a  tls listener instead of a plaintext http listener. Wouldn't it be easier to use a proper url for the CdServer variable like this `http://kuberpult-cd-service:8443` -> plaintext, `https://kuberpult-cd-service:8443` ->  tls?

Reply:
> We will need this on the rollout service as well.

You are right. I will add this to the rollout-service as well.

> Also, I'm not too happy with the config name "TlsEnabled" could also mean that the http server creates a tls listener instead of a plaintext http listener.

Yes, this could be confusing. How about `GrpcTlsEnabled`?

> Wouldn't it be easier to use a proper url for the CdServer variable like this `http://kuberpult-cd-service:8443` -> plaintext, `https://kuberpult-cd-service:8443` -> tls?

AFAIK, grpc expects the `domain:port` without the protocol. If you supply the protocol as well, you get `too many colons in address` error.

Reply:
Yes,  grpc expects the host without protocol. However, we can easily extract this from the url ourself.

Reply:
As agreed, I updated the name of the environment variable.